src/webRTCEndpoint.ts

```diff
@@ -702,7 +703,8 @@ export class WebRTCEndpoint extends (EventEmitter as new () => TypedEmitter<Requ
       this.connection &&
       (this.connection.signalingState !== "stable" ||
         this.connection.connectionState !== "connected" ||
-        this.connection.iceConnectionState !== "connected")
+        this.connection.iceConnectionState !== "connected" ||
+        this.ongoingTrackReplacement)
```

Comment:
I would suggest moving that to previous `if`, and leave this guard just for `this.connection`.